package.json

```diff
@@ -380,9 +380,14 @@
     "@microsoft/api-extractor/typescript": "5.3.3",
     "swc-loader": "^0.2.6",
     "prettier": "2.8.8",
-    "puppeteer": "19.6.0",
+    "puppeteer": "19.6.3",
     "ws": "8.17.1",
-    "**/@mdx-js/react/@types/react": "^17"
+    "**/@mdx-js/react/@types/react": "^17",
+    "**/prismjs": "^1.30.0",
+    "**/@tensile-perf/runner/express": "^4.21.2",
+    "**/tar-fs": "^2.1.2",
```

Comment:
undo once we migrate to latest puppeteer - for now its out of scope to do the upgrade